File: PHP Excel Examples/WorksheetsApiDemo/AutofitRowsColumns/autofitColumns.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;
use Spire\Cloud\Excel\Sdk\Model\AutoFitterOptions;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "AutofitColumns.xlsx";
$folder = "input";
$storage = "";
$sheetName = "sheet5";
$firstColumn = 1;
$lastColumn = 5;
$autoFitterOptions = new AutoFitterOptions(null);
$autoFitterOptions->setAutoFitMergedCells(true);
$autoFitterOptions->setIgnoreHidden(true);
$autoFitterOptions->setOnlyAuto(true);
$firstRow = 3;
$lastRow = 10;
$result = $apiInstance->autofitColumns($name, $sheetName, $firstColumn, $lastColumn, $autoFitterOptions, $firstRow, $lastRow, $folder, $storage);

File: Python Excel Examples/WorksheetsApiDemo/auto_fit_rows_columns/autofitColumns.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "AutofitColumns.xlsx"
password = ""
storage = ""
folder = "ExcelDocument"
sheet_name = "Sheet1"
first_column = 1
last_column = 5
auto_fitter_options = spirecloudexcel.models.AutoFitterOptions(True, True, True)
first_row = 3
last_row = 10
api.autofit_columns(name, sheet_name, first_column, last_column, auto_fitter_options, first_row=first_row,last_row=last_row, storage=storage, folder=folder)